### DISCUSSION TITLE
Attu User/Roles management

### DISCUSSION THREAD

**Discussion by navneetrajput06** [milvus-io/milvus#27655] [Q&A and General discussion]
Can we create a specific attu user access. eg. a user should only be able to view particular db and the data under the hood of that db

**Comment by yhmo** [comment]
Actually, the attu user is milvus user/role: https://milvus.io/docs/rbac.md
1. create a user in milvus
2. specify privilege to the user
3. login attu with the user/password.

**Reply by navneetrajput06** [reply]
Hi Team,
I tried to follow this doc but its not for DB specific. we need to apply rules on DB. so that user can view only a particular DB.

**Reply by yhmo** [reply]
Currently, milvus only has two levels of privileges control: Collection and Global

Global privileges will take effect in all databases.

ShowCollection is a Global-level privilege, if you assign it to a user, the user can see all the collections in all the databases.

Only the Collection level privileges can be controlled within a database.
